test/entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh -l
+
+check_return_code () {
+   if [ $? -eq 0 ]; then
+     echo PASS
+   else
+     echo FAIL
+     exit 1
+   fi
+}
+
+#Clair DATA
+curl -H "Content-type: application/json" -XGET https://quay.io/api/v1/repository/redhat-appstudio/service-provider-integration-scm-file-retriever-server/manifest/sha256:b4449fd78ddd191995bc311071a0dee89bbdb442e7191f86b908982d0727b5a6/security?vulnerabilities=true > clair.json
```

Comment:
We've been having discussions with the quay.io/Clair team and the key takeaway is that Clair vulnerabilities are added on a regular basis as the vulnerability database gets updated with new ones.
This unfortunately means that any image that has no vulnerabilities today might have them tomorrow.

In light of this, I think we'll have to go for a negative test for this one. I suggest using the `quay.io/redhat-appstudio/sample-image:test-index-fail` for this: 
```
https://quay.io/api/v1/repository/redhat-appstudio/sample-image/manifest/sha256:fc78d878b68b74c965bdb857fab8a87ef75bf7e411f561b3e5fee97382c785ab/security?vulnerabilities=true
```

Reply:
Done. I was actually asking Jiri about the same issue before and he has provided me with another test data for time being.